content/support.md

```diff
@@ -18,20 +18,28 @@ While we strive for "best effort" with regard to non-deployment questions pertai
 
 If you would like expert assistance with your Meteor question, consider a subscription to [Meteor Developer Support](#developer).
 
-Alternatively, consider using <a href="http://github.com/meteor/meteor/issues/">GitHub</a> or the <a href="https://forums.meteor.com/">forums</a>. GitHub is excellent for tracking issues, while the forums are provided for community use concerning any kind of issue.
-
-<h3 id="Enhanced Galaxy Support Options">Enhanced Galaxy Support Options</h3>
+<h3 id="enhanced-galaxy-support">Enhanced Galaxy Support Options</h3>
 
 Galaxy customers have the option to upgrade to higher levels of paid Galaxy support for guaranteed response times as short as 15-minutes with 24x7x365 coverage, including critical cases related to application downtime or degraded performance.
 
 Please contact [Galaxy Sales](mailto:galaxysales@meteor.com) for more details.
 
-<h3 id="Meteor Developer Support">Meteor Developer Support</h3>
+<h3 id="meteor-developer-support">Meteor Developer Support</h3>
 
 Meteor Developer Support helps customers build production Meteor apps the right way, ensuring architectural best practices and timely responses to technical inquiries related to Meteor application development.
 
 Please fill out our <a name="developer"/>[Developer Support](https://www.meteor.com/developer-support)</a> form for more information.
 
+<h3 id="Scope">Support Scope</h3>
+
+Because of the wide surface area involved with Galaxy, a number of issues peripherally connected to Galaxy may be classified as out of scope. Questions about an app's performance, stability and implementation are separate from, and different than, supported questions targeting Galaxy's uptime, health, and general deployment capabilities.
+
+Unsupported issues would include a conflict between a third-party npm package and Galaxy that prevents deployment, a database issue on a Galaxy-hosted app, or a question about a logged error that needs to be resolved at the Meteor code level. If you write in to support about an out of scope issue, Galaxy support will explain its status and decline to continue investigating the issue. 
+
+If  an enhanced support experience that does cover these use cases is essential for your app, consider signing up for [Meteor Developer Support](#meteor-developer-support) or [Enhanced Galaxy Support](#enhanced-galaxy-support).
```

Comment:
In this section, just mention Meteor Developer Support, and not the critical SLA Galaxy Support options.

This line would then be: "To get support for debugging Meteor application code and architecture issues, consider signing up for Meteor Developer Support"